CMakeLists.txt

```diff
@@ -27,6 +27,16 @@ cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
 
 project(pocl)
 
+set(LATEST_KNOWN_CPP_VERSION "20")
```

Comment:
Please use CXX for C++. CPP is the preprocessor. A better name would be LATEST_KNOWN_CXX_STD_VERSION.